### PR TITLE
Feat/dc 807 update read me for gamma depth app

### PR DESCRIPTION
**Pull request by nickbrinser**
- Updating readme with app details

### CODE REVIEW

File: apps/sample_gamma_depth/README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# Getting Started with Create Corva App
+# Sample Gamma Depth
+
+## Project Overview
+
+In this project our goal is to display Depth vs Gamma Ray data for a specific asset. This version of the app fetches data from a static collection by asset id. Future versions of this app will be subscribed to a dataset for continuous live updates.
```
Comment (dumavit) [inline apps/sample_gamma_depth/README.md:5]: asset id may be ambiguous to some users. let's use well id?

File: apps/sample_gamma_depth/README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+
+This data is then formatted into an accepted Highcharts data format which can be an array of cardinal points as either `[x, y]` or `{name, x, y}`. For out use case we're simple using the former x being our measured depth value and y being our gamma ray value.
```
Comment (dumavit) [inline apps/sample_gamma_depth/README.md:7]: for OUR use case we're SIMPLY

File: apps/sample_gamma_depth/README.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+
+To make changes to the chart, you would simple update the options object in the GammaChart component. Here you can edit the the chart's type, appearance, and series data.
```
Comment (dumavit) [inline apps/sample_gamma_depth/README.md:9]: the the the :)

File: apps/sample_gamma_depth/README.md (hunk 1 above)
Comment (nickbrinser) [reply apps/sample_gamma_depth/README.md:5]: Won't this cause confusion for non-corva devs since we use `well.asset_id` for certain api calls etc?

**Pull request merged**